assets/css/form.css

```diff
@@ -47,10 +47,26 @@ header:after {
   visibility: hidden;
 }
 
+.required {
```

Comment:
@xtine Were these changes intentional? Asking because they're not content changes, not because they don't look good. :)